docs/swagger/api.yaml

```diff
@@ -233,6 +248,11 @@ definitions:
                 format: double
                 description: Price of the item
                 example: 1.0
+              vat:
+                type: number
+                format: double
+                description: VAT
+                example: 20.0
```

Comment:
so we've got : 
```
              price:
                type: number
                format: double
                description: Price of the item
                example: 1.0
              vat:
                type: number
                format: double
                description: VAT
                example: 20.0
```

Like that is not very clear what is the price - price with vat or without vat

Reply:
The price would not change based on the vat.